### ISSUE TITLE
Implement auth from ZeroMQ 4

### ISSUE THREAD

**Issue by asvetlov** [aio-libs/aiozmq#16]

**Comment by narodnik** [comment]
Has there been any progress on this issue? I cannot find anything in the docs about auth or curve support.

**Comment by JelleZijlstra** [comment]
No. I'm currently the only maintainer and I'm not planning to work on any new features, just on making sure the library keeps working with new Python versions (#166). I'd be happy to review PRs though.
